pallets/ddc-customers/src/lib.rs

```diff
@@ -67,12 +67,9 @@ pub struct Bucket<T: Config> {
 	cluster_id: ClusterId,
 	is_public: bool,
 	is_removed: bool,
-	total_customers_usage: Option<CustomerUsage>,
-}
-
-#[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
-pub struct BucketParams {
-	is_public: bool,
+	// todo(yahortsaryk): `total_customers_usage` should be renamed to `total_usage` to eliminate
```

Comment:
This would be a big change that affects too many integrations and dependencies. I'm not sure we should merge `ddc-customers` to another module. It may be better to split it into `ddc-buckets` and `ddc-customers` or smth. else. Anyway, it's a topic for a separate discussion